### ISSUE TITLE
Should we have `run_exports` on `LinkingTo` packages?

### ISSUE THREAD

**Issue by mfansler** [conda-forge/r-rcpp-feedstock#39]
<sub>Note: This isn't specifically an Rcpp issue, but Rcpp is a prominent representative of the class of packages to which this pertains.</sub>

Browsing issues on Bioconda, I noticed that [there is some effort](https://github.com/bioconda/bioconda-recipes/issues/43905) to more broadly add `run_exports` to their recipes, including for Bioconductor recipes. While I'm not so sold on putting it on everything, I am wondering:

**Should we be adding `run_exports` to packages that get used in `LinkingTo` sections?**

Personally, I've never run across an issue reporting having hit an Rcpp runtime incompatibility, but who knows if a user would know where to report it if encountered.

Any thoughts, @conda-forge/r?

**Comment by cbrueffer** [comment]
Indeed, I don't think I've seen issues either, but who knows. I don't have strong opinions on this either way, but would lean to keeping things as is for now. In Bioconda we have a lot lot more varied ecosystem where incompatibility issues are probably more likely to crop up.

**Comment by mfansler** [comment]
I just hit one of these with `r-lme4` linking against `r-matrix` (https://github.com/conda-forge/r-matrix-feedstock/issues/42). However, still doesn't seem worth using run exports.
